mzLib/Chemistry/ChemicalFormula.cs

```diff
@@ -523,5 +525,37 @@ private string GetHillNotation()
             otherParts.Sort();
             return s + string.Join("", otherParts);
         }
+
+        public override string ToString()
+        {
+            return $"{ThisChemicalFormula.Formula} : {MonoisotopicMass}";
+        }
+
+        public static ChemicalFormula operator -(ChemicalFormula left, IHasChemicalFormula right)
+        {
+            if (left == null)
+                return right == null ? null : new ChemicalFormula(right);
```

Comment:
For subtraction operations,  should you throw an exception if you end up with a negative chemical formula? It looks like null - H20 would return H20, but is that the intended behavior?

Reply:
not sure about this. we use negative chemical formulas in mods. For example C-8H-5N-1

Reply:
This wouldn't return negative values  if you subtract something from a null value. Is that something that needs to happen?